content/kubeone/master/architecture/operating-system-manager/usage/_index.en.md

```diff
@@ -147,6 +147,9 @@ spec:
             data: echo "hello world"
 ```
 
+{{% notice warning %}} Make sure to escape all occurrences of Go text template in your custom OperatingSystemProfiles. For example, if you want have "{{ .Token }}" in your OSP then it should be replaced with "{{ \`{{ .Token }}\` }}".
```

Comment:
```suggestion
{{% notice warning %}}

Make sure to escape all occurrences of Go text template in your custom
OperatingSystemProfiles. For example, if you want have `{{.Token}}` in your OSP
then it should be replaced with ```{{ `{{.Token}}` }}```.

{{% /notice %}}
```

Not marking it as a code looks a bit meh.
![image](https://user-images.githubusercontent.com/30984/187899272-26392313-b48b-4e96-aa2e-275dd5e36192.png)

New look:
![image](https://user-images.githubusercontent.com/30984/187900565-e716dabe-cdbf-496e-b5ae-11431dcabdc1.png)

Reply:
Awesome, I tried doing this and failed, so just gave up(too early :P) :D